application/src/main/java/org/thingsboard/server/service/lwm2m/LwM2MServiceImpl.java

```diff
@@ -50,22 +51,26 @@ private ServerSecurityConfig getServerSecurityConfig(LwM2MSecureServerConfig ser
         bsServ.setPort(serverConfig.getPort());
         bsServ.setSecurityHost(serverConfig.getSecureHost());
         bsServ.setSecurityPort(serverConfig.getSecurePort());
-        bsServ.setServerPublicKey(getPublicKey(serverConfig));
+        byte[] publicKeyBase64 = getPublicKey(serverConfig);
+        if (publicKeyBase64 == null) {
+            bsServ.setServerPublicKey("");
+        } else {
+            bsServ.setServerPublicKey(Base64.encodeBase64String(getPublicKey(serverConfig)));
```

Comment:
use publicKeyBase64 instead of getPublicKey(serverConfig)

Reply:
OK